src/components/Table/Table.test.jsx

```diff
@@ -334,4 +334,48 @@ describe('Table', () => {
     fireEvent.keyDown(filterButton, { keyCode: keyCodes.ENTER });
     expect(mockActions.toolbar.onToggleFilter).toHaveBeenCalledTimes(1);
   });
+
+  test('toolbar search should render with default value', () => {
+    const wrapper = mount(
+      <Table
+        columns={tableColumns}
+        data={tableData}
+        actions={mockActions}
+        options={{
+          hasSearch: true,
+        }}
+        view={{
+          toolbar: {
+            search: {
+              defaultValue: 'toyota',
+            },
+          },
+        }}
+      />
+    );
+
+    expect(wrapper.find('.bx--search-input')).toHaveLength(1);
+    expect(wrapper.find('.bx--search-input').prop('value')).toEqual('toyota');
+
+    const wrapper2 = mount(
```

Comment:
This test is basically the same as the one above it... ie, create a table and verify the value exists (which is a good test, but no need to do it twice).  A better test here for this second one would be use the first wrapper and update the props and then see if the that table instance now reflects the new default search value as set from changing the props.

The use case for how this used is as follows.
We use the table to render the state of a "datasource" which is basically some stateful component that holds things like our data, responds to searches, etc.  That table instance could dynamically load a new "view" which means that it might have a default search, etc, and when that happens we need to the UI to reflect the state of our datasource.  Today, everything about the table, including filters, columns, etc, all comes from our datasource, and it all works woo, with the exception that the "search" value cannot be set in the toolbar.  So we load a new view and show it, and the data is filtered, but, the UI doesn't show that search value.  In essence we have a single table and we just keep updating the props on that table to reflect the state of the data from the datasource.

Reply:
Gotcha! The duplication was just to make sure aliasing works but, given `value` is being deprecated, I removed it and put something like this in place: https://github.com/IBM/carbon-addons-iot-react/pull/953/commits/f16851c5bbdabe46d120327146f41c11308e5871#diff-f47894e32d4dbf684d2ac3f89df148c9R360